src/client/components/ModalNew.tsx

```diff
@@ -130,7 +130,8 @@ export const Modal: React.FC<ModalProps> = ({
         variants={{
           visible: {
             opacity: 1,
-            transform: 'translateX(-50%) translateY(-50%) scale(1)',
+            x: '-50%',
```

Comment:
I was under the impressions that `transform` would be GPU accelerated :) But then again, if it's a lot of layers then it's not always so smooth either. Does it show what css properties `framer-motion` animates when using x and y?

Reply:
Yeah, apparently all you should need to do is include `translateZ(0)` to make GPU acceleration kick in but this doesn't happen in Safari... Anyways, I just updated the syntax since it's easier to understand :) Or what do you think?

Reply:
Yeah, I assume framer-motion does other magic as well so it might not matter if you use x,y or transform. So if this performs better, let's merge! 🙌